source/index.rst

```diff
@@ -33,6 +33,7 @@ Welcome to the *FIRST*\ |reg| Robotics Competition Control System Documentation!
    :caption: Programming Basics
 
    docs/software/what-is-wpilib
+   docs/software/frc-glossary
```

Comment:
Thinking about it, maybe we should move this low down in the index. I don't think we want people to actively read it, but more use it as a page to reference for ``:term:``

Reply:
makes sense